plugins/CorePluginsAdmin/Controller.php

```diff
@@ -144,8 +146,52 @@ public function browseThemes()
         $this->redirectToIndex('Marketplace', 'overview', null, null, null, array('show' => 'themes'));
     }
 
+    public function tagManagerTeaser()
+    {
+        $this->dieIfPluginsAdminIsDisabled();
+        Piwik::checkUserHasSomeAdminAccess();
+
+        $tagManagerTeaser = new TagManagerTeaser(Piwik::getCurrentUserLogin());
+
+        if (!$tagManagerTeaser->shouldShowTeaser()) {
+            $this->redirectToIndex('CoreHome', 'index');
+            return;
+        }
+
+        $nonce = '';
+        if (Piwik::hasUserSuperUserAccess()) {
+            $nonce = Nonce::getNonce(static::ACTIVATE_NONCE);
+        }
+
+        $superUsers = Request::processRequest('UsersManager.getUsersHavingSuperUserAccess', [], []);
+        $emails = implode(',', array_column($superUsers, 'email'));
+
+        $view = new View('@CorePluginsAdmin/tagManagerTeaser');
+        $this->setGeneralVariablesView($view);
+        $view->superUserEmails = $emails;
+        $view->nonce = $nonce;
+        return $view->render();
+    }
+
+    public function disableActivateTagManagerPage()
+    {
+        $this->dieIfPluginsAdminIsDisabled();
+        Piwik::checkUserHasSomeAdminAccess();
+
+        $tagManagerTeaser = new TagManagerTeaser(Piwik::getCurrentUserLogin());
+
+        if (Piwik::hasUserSuperUserAccess()) {
+            $tagManagerTeaser->disableGlobally();
```

Comment:
Is it possible for a superuser to want to disable the tag manager for themselves only?

Reply:
nope, cause then it wouldn't be possible anymore to disable it for all users.

Reply:
what i mean is, if one superuser might want to disable it for all users, while another might want to just disable it for themselves. ie, should the feature to disable all be separate from the feature to disable for the current logged in user (super user or not)

Reply:
Gotcha